AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml

```diff
@@ -75,24 +82,44 @@
-        <script src="~/lib/jquery/dist/jquery.js"></script>
-        <script src="~/lib/bootstrap/dist/js/bootstrap.js"></script>
-        <script src="~/lib/bootstrap-toggle/js/bootstrap-toggle.js"></script>
-        <script src="~/lib/knockout/dist/knockout.js"></script>
-        <script src="~/lib/moment/moment.js"></script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.js"></script>
+        <script src="//ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.4.js"
+                asp-fallback-src="~/lib/jquery/dist/jquery.js">
+        </script>
+        <script src="//ajax.aspnetcdn.com/ajax/knockout/knockout-3.3.0.js"></script>
+        <script src="//ajax.aspnetcdn.com/ajax/bootstrap/3.0.0/bootstrap.js"
+                asp-fallback-src="~/lib/bootstrap/dist/js/bootstrap.js">
+        </script>
+        <script src="//gitcdn.github.io/bootstrap-toggle/2.2.0/js/bootstrap-toggle.min.js"
+                asp-fallback-src="~/lib/bootstrap-toggle/js/bootstrap-toggle.js">
+        </script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.js"
+                asp-fallback-src="~/lib/moment/moment.js">
+        </script>
         <script src="~/lib/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js"></script>
-        <script src="~/lib/tota11y/build/tota11y.js"></script>
-        <script src="~/lib/tinymce/tinymce.js"></script>
-        <script src="~/lib/bootstrap-tagsinput/dist/bootstrap-tagsinput.js"></script>
-        <script src="~/lib/d3/d3.js"></script>
-        <script src="~/lib/jquery.maskedinput/dist/jquery.maskedinput.js"></script>
-        <script src="~/lib/fullcalendar/dist/fullcalendar.js"></script>
+        <script src="//ajax.aspnetcdn.com/ajax/bootstrap-touch-carousel/0.8.0/js/bootstrap-touch-carousel.js"
+                asp-fallback-src="~/lib/bootstrap-touch-carousel/dist/js/bootstrap-touch-carousel.js">
+        </script>
+        <script src="//cdn.tinymce.com/4/tinymce.js"
+                asp-fallback-src="~/lib/tinymce/tinymce.js">
+        </script>
+        <script src="//d3js.org/d3.v3.js"
+                asp-fallback-src="~/lib/d3/d3.js">
+        </script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.maskedinput/1.4.1/jquery.maskedinput.js"
+                asp-fallback-src="~/lib/jquery.maskedinput/dist/jquery.maskedinput.js">
+        </script>
+
+        <script src="//cdnjs.cloudflare.com/ajax/libs/fullcalendar/2.9.1/fullcalendar.min.js"
+                asp-fallback-src="~/lib/fullcalendar/dist/fullcalendar.js">
+        </script>
     </environment>
     <environment names="Staging,Production">
         <script src="//ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.4.min.js"
                 asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
                 asp-fallback-test="window.jQuery">
         </script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.js"></script>

```

Comment:
Needs `asp-fallback-src` and `asp-fallback-test` attributes. See the CDNs and Fallbacks section here: http://www.davepaquette.com/archive/2015/05/06/link-and-script-tag-helpers-in-mvc6.aspx